variables.tf

```diff
@@ -50,3 +50,9 @@ variable "additional_cidr_blocks" {
   description = "A list of additional IPv4 CIDR blocks to associate with the VPC"
   default     = null
 }
+
+variable "ipv6_cidr_block" {
```

Comment:
IMHO, I think this variable name would be better as `assign_ipv6_cidr_block` to better reflect what it is doing. Even `assign_generated_ipv6_cidr_block` to be really clear.

Reply:
Variable name has been changed.